packages/ui/components/ui/tx/view/action-view.tsx

```diff
@@ -3,6 +3,38 @@ import { SpendViewComponent } from './spend';
 import { OutputViewComponent } from './output';
```

Comment:
The changes in this file are totally optional, and aren't strictly related to what the PR is about. But I found myself referring back to this component a lot, and realized it was a bit easier to understand if rewritten this way. That said, that could just be me, and I'm happy to revert this change if it's undesirable.